CMakeLists.txt

```diff
@@ -264,7 +264,9 @@ if (USE_STATIC_LIBRARIES AND HAVE_NO_PIE)
     set (CMAKE_C_FLAGS                   "${CMAKE_C_FLAGS} ${FLAG_NO_PIE}")
 endif ()
 
-if (NOT SANITIZE)
+# TODO: only make this extra-checks in CI builds, since a lot of contrib libs won't link -
+#       CI works around this problem by explicitly adding GLIBC_COMPATIBILITY flag.
```

Comment:
Why we can't do it always?

Reply:
Because @proller complains that he can't build locally otherwise, and don't want to use extra flag as workaround.

Reply:
Ok.

Reply:
Also it needed for arcadia